### PR TITLE
[#1] Add BOM, Inspector Axon, IdeaPlugin, and Initializr sections

### LINKED ISSUE #1: Explain all projects under the Axon Framework organization
The main `README.md` should reference all the repositories present in the [Axon Framework](https://github.com/AxonFramework/) GitHub organization.
Furthermore, a reference to the documentation and code samples should also be included.

Thus, this includes:
- Axon Framework
- Documentation
- Samples
- Extensions
- BOM
- Intellij Plugin

### PR DESCRIPTION
**Pull request by smcvb**
This pull request adds the following sections to the `README.md`:

- Bill of Materials - the BOM ensuring compatible versions are used of AF-specific projects
- Inspector Axon - tool to enhance the AF experience of any user
- IntelliJ IDEA Plugin - plugin to IntelliJ to ease AF development
- AxonIQ Initializr - web application enabling users to kickstart an AF+SpringBoot-based application

In doing so, this issue resolves #1, as #2 and #5 provided the rest of #1.

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -271,8 +271,47 @@ Please read the [Tracing section](https://docs.axoniq.io/reference-guide/extensi
 
 ## Bill of Materials
 
+The [Bill of Materials](https://github.com/AxonFramework/axon-bom) repository of Axon Framework is a dedicated dependency providing compatible versions of the [framework](#axon-framework) and [extensions](#extensions).
+Hence, by adding `axon-bom` to your dependency management system, you ensure the compatibility of Axon's dependencies.
+As such, we recommend you use this dependency whenever you combine Axon Framework with any of the extensions.
+
+The BOM is intended to release with every [Axon Framework](https://github.com/AxonFramework/AxonFramework) release.
```
Comment (CodeDrivenMitch) [inline profile/README.md:278]: Shouldn't we add a caveat that the versions might differ? This text really seems like we keep them aligned, and If I recall our discussion correctly, we have no intention/means of doing that

File: profile/README.md [hunk 2]
```diff
@@ -271,8 +271,47 @@ Please read the [Tracing section](https://docs.axoniq.io/reference-guide/extensi
...
+Hence, a 4.7.0 release of Axon Framework leads to a 4.7.0 release of the BOM.
+
 ## Inspector Axon
 
+The Axon Framework team designed [Inspector Axon](https://inspector.axoniq.io/) to further enable [Axon Framework](#axon-framework) applications with valuable data and tooling.
```
Comment (CodeDrivenMitch) [inline profile/README.md:283]: 🥳 🥳 🥳 🥳 🥳 🥳 🥳 🥳

File: profile/README.md [hunk 3]
```diff
@@ -271,8 +271,47 @@ Please read the [Tracing section](https://docs.axoniq.io/reference-guide/extensi
...
+By doing all this, we aim to teach the inner workings of Axon Framework to any user, enabling you the get the most out of the framework.
+
+The [how it works](https://inspector.axoniq.io/how-it-works) page helps you understand how Inspector Axon provides all this information.
+We refer you to [this](https://inspector.axoniq.io/legal) page if you (and your team) are curious to understand how we secure all this behavior.
+
 ## IntelliJ IDEA Plugin
 
-## AxonIQ Initializr
\ No newline at end of file
+The [Axon Framework IntelliJ Plugin](https://github.com/AxonFramework/IdeaPlugin) is a plugin you can add when using [IntelliJ IDEA](https://www.jetbrains.com/idea/).
```
Comment (CodeDrivenMitch) [inline profile/README.md:298]: Should we link to the marketplace directly? https://plugins.jetbrains.com/plugin/18628-axon-framework

**Review by CodeDrivenMitch — APPROVED** [review]
Looks good to me, apart from two small comments

File: profile/README.md (hunk 3 above)
Comment (smcvb) [reply profile/README.md:298]: Super smart! Will do

**Pull request merged**